### PR TITLE
Finnish translation & grammar update

### PR DESCRIPTION
**Pull request by valtsu41**

### CODE REVIEW

**Comment by valtsu41** [comment]
Still WIP but I opened this to prevent merge conflicts in the future.

**Comment by valtsu41** [comment]
@xzarfs are these changes ok?

**Comment by valtsu41** [comment]
@xzarfs doesn't seem to be answering, should I put this as mergable or wait?

**Comment by valtsu41** [comment]
Whatever, I'll just open it.

**Comment by xzarfs** [comment]
Sorry for being this late.
You made alot of great improvements . There are few unclear things in your commits thought, also and I think capitalizing every word in a short sentance looks good in the gui (and makes translations seem more consistant ) for e.g. Tallenna kaavio -> Tallenna Kaavio

**Comment by valtsu41** [comment]
Well, the thing with that is that the official way to do that is only first letter uppercase.

**Comment by valtsu41** [comment]
Can you leave line comment on the unclear things?

**Comment by xzarfs** [comment]
Made a few line comments. Its all right after those get checked. 

About the word case, its okay done eather ways.

**Comment by valtsu41** [comment]
Wait, how do I see them?

**Comment by xzarfs** [comment]
can you see them now?

**Comment by valtsu41** [comment]
Yes, I'll try fixing those.

File: core/assets/bundles/bundle_fi.properties [hunk 1]
```diff
@@ -308,29 +308,29 @@ editor.waves = Tasot:
...
 waves.invalid = Invalid waves in clipboard.
-waves.copied = Aallot kopioitu.
+waves.copied = Tasot kopioitu.
 waves.none = No enemies defined.\nNote that empty wave layouts will automatically be replaced with the default layout.
 editor.default = [lightgray]<Default>
 details = Yksityiskohdat...
 edit = Muokkaa...
 editor.name = Nimi:
-editor.spawn = Spawni yksikkö
+editor.spawn = Luo yksikkö
```
[the remark this answers is not in the document]
Comment (valtsu41) [reply core/assets/bundles/bundle_fi.properties]: From what I remember, that button *spawns an unit* and not a spawn point.

[the remark this answers is not in the document]
Comment (valtsu41) [reply core/assets/bundles/bundle_fi.properties]: (Though I think it is not used anymore)

**Comment by valtsu41** [comment]
Is that ok?

**Comment by xzarfs** [comment]
Its great. Hope this gets merged soon.

**Pull request merged**